extensions/wikia/GlobalNavigation/scripts/GlobalNavigationSearch.js

```diff
@@ -2,6 +2,7 @@ require(['jquery', 'wikia.browserDetect', 'GlobalNavigationiOSScrollFix'], funct
 	'use strict';
 	var $selectElement = $('#searchSelect'),
```

Comment:
please move assignment to those variables to function which is called after DOM loaded in order to avoid race conditions.